csrankings-a.csv

```diff
@@ -1390,6 +1390,7 @@ Andreas Galanis,University of Oxford,https://www.cs.ox.ac.uk/people/andreas.gala
 Andreas Geiger 0001,University of Tübingen,http://www.cvlibs.net,SrVnrPcAAAAJ
 Andreas Goerdt,TU Chemnitz,https://www.tu-chemnitz.de/informatik/TI/professur/professor.php,NOSCHOLARPAGE
 Andreas Haeberlen,University of Pennsylvania,http://www.cis.upenn.edu/~ahae,kqPhodQAAAAJ
+Andreas Hein,University of Oldenburg,https://uol.de/amt/mitarbeiter/andreas-hein,NOSCHOLARPAGE

```

Comment:
Not in CS. Please clarify.

Reply:
Prof. Dr. -Ing. Andreas Hein is the director of the division "Assistenzsysteme und Medizintechnik" "Assistance systems and Medical Technology" (AMT), which has moved from the computer science department to the healthcare research department in 2013, already: https://uol.de/amt Still, he has a close connection to the computer science department. There is overlap in the used technologies like robotics, virtual reality and machine learning. For example, both his current project groups are listed not only for the "engineering of sociotechnical systems" and the "embedded systems and microrobotics" course of study, but also for the "Informatik" (Computer Science) and "Wirtschaftsinformatik" (business informatics) courses of study:

![image](https://github.com/emeryberger/CSrankings/assets/11575/74a0ff87-5b17-4c28-aa76-55161b4f74f1)

As Prof. Hein is a permanent faculty member of a non-CS department who can advise PhD students in computer science, he should meet the inclusion criteria, or am I missing something?

Reply:
If he can *solely* advise students for a PhD *in Computer Science*, he is eligible. If not, not.

Reply:
Thanks for clarifying. As he can not solely, but also advise students for a PhD in Computer Science, I removed Andreas Hein from this file.